tests/ui/README.md

```diff
@@ -0,0 +1,91 @@
+
+Getting involved
+----------------
+
+We love working with contributors to fill out the Selenium test coverage for addons-frontend,
+but it does require a few skills.
+By contributing to our test suite you will have an opportunity to learn and/or improve your
+skills with Python, Selenium WebDriver, GitHub, Virtualenv, the Page Object Model, and more.
+
+For some resources for learning about these technologies, take a look at our documentation on
+[running Web QA automated tests][running-tests].
+
+All of [these awesome contributors][contributors] have opened pull requests
+against this repository.
+
+
+Questions are always welcome
+----------------------------
+While we take pains to keep our documentation updated, the best source of information is those
+of us who work on the project.
+Don't be afraid to join us in irc.mozilla.org [#mozwebqa][mozwebqa] to ask questions about our
+Selenium tests.
+We also have a [mailing list][mailing_list] available that you are welcome to join and post to.
+
+How to run the tests locally
+-----------------------------------------
+We maintain a [detailed guide][running-tests] to running our automated tests which can be found on the MDN website.
+If you want to get started quickly, you can try following the steps in our quick-start instructions below:
+
+### Clone the repository
+If you have cloned this project already then you can skip this, otherwise you'll need to clone this repo using Git.
+If you do not know how to clone a GitHub repository, check out this
+[help page][git-clone] from GitHub.
+
+If you think you would like to contribute to the tests by writing or maintaining them in the future,
+it would be a good idea to create a fork of this repository first, and then clone that.
+GitHub also has great instructions for [forking a repository][git-fork].
+
+### Create or activate a Python virtual environment
+You should install this project's dependencies (which is described in the next step) into a virtual environment
+in order to avoid impacting the rest of your system, and to make problem solving easier.
+If you already have a virtual environment for these tests, then you should activate it,
+otherwise you should create a new one.
+For more information on working with virtual environments see our
+[summary][virtualenv].
+
+### Install dependencies
+Install the Python packages that are needed to run our tests using pip. In a terminal,
+from the the project root, issue the following command:
+
+```bash
+$ pip install -Ur requirements.txt
+```
+
+### Run the tests
+Tests are run using the command line. Below are a couple of examples of running the tests:
+
+To run all of the desktop tests against the default environment, which is the discopane staging environment:
+
+Note: The tests must be run in Firefox 48 or later.
+
+```bash
+$ py.test --driver Firefox --firefox-path=<path-to-firefox> --driver-path=<path-to-wires-executable> test_discopane.py
```

Comment:
These tests only make sense to run against Firefox, so let's make it the default. Add a `setup.cfg` with a `[pytest]` section and add the line `addopts=--driver=Firefox` then you can remove this from the example command. See [here](https://github.com/davehunt/cookiecutter-pytest-selenium/blob/master/%7B%7Bcookiecutter.slug%7D%7D/setup.cfg#L2) for an example.